package/yast2-ntp-client.spec

```diff
@@ -39,11 +39,11 @@ BuildRequires:  rubygem(%rb_default_ruby_abi:yast-rake)
 Requires:       augeas-lenses
 Requires:       yast2 >= 3.2.21
 Requires:       yast2-country-data
+# needed for network/config agent
+Requires:       yast2-network
```

Comment:
as it seems we should move the agent into yast2.rpm

Reply:
:+1:

Reply:
but I would like to not slow down this pr, so can be done separately.